manual/sphinx/user_docs/advanced_install.rst

```diff
@@ -213,37 +213,33 @@ The following configuration commands have been used successfully:
     make
     make install
 
-* netCDF-4.7.4::
+* netCDF-4.7.4 (note: using cmake to build gave errors in ``make test`` while the autotools build with ``configure`` passed all tests in ``make check``; netCDF-4.8.0 failed to link to hdf5-1.12.0 and failed tests with both cmake and autotools)::
```

Comment:
I think it is nicer to put the notes below the commands, without `()` - but this is just personal taste.

Reply:
both good points, but it's merged now...

Reply:
Sorry, I was a bit quick off the mark there!